assets/scss/layouts/_header.scss

```diff
@@ -168,8 +168,7 @@ button#doks-versions {
 
   .doks-search {
     max-width: 20rem;
-    margin-top: 0.125rem;
-    margin-bottom: 0.125rem;
+    margin: 0 auto;
```

Comment:
This change is unrelated to the PR title and blocked by a question in https://github.com/BluBracket/docs.blubracket.com/pull/97#discussion_r821314162

Reply:
I dropped this commit from this PR as well.